engine/src/cubos/engine/collisions/broad_phase/plugin.cpp

```diff
@@ -0,0 +1,199 @@
+
+#include "plugin.hpp"
+
+#include <cubos/engine/collisions/broad_phase/candidates.hpp>
+#include <cubos/engine/collisions/collider.hpp>
+#include <cubos/engine/collisions/shapes/box.hpp>
+#include <cubos/engine/collisions/shapes/capsule.hpp>
+#include <cubos/engine/transform/local_to_world.hpp>
+
+#include "sweep_and_prune.hpp"
```

Comment:
Why are they away from each other?

Reply:
because auto formatter

Reply:
Thats weird, I think that if you put them together (`"plugin.hpp"` and `"sweep_and_prune.hpp"` to clarify) and format they should stay together. Try doing that.

Reply:
Tried again just to confirm but nope. The header order is:
1. Header corresponding to source file
2. STL headers
3. Include headers
4. Local headers

Reply:
Ahh makes sense, ty. Including our own header first is good practice, but we end up not doing that with public headers because we include them with angle brackets. Maybe we should consider using quotes there?

Reply:
Would it even work with quotes? public headers would need tons of `../` lmao. I don't think there's a way to avoid this

Reply:
Thats true :pensive: